src/extensions/default/NoDistractions/main.js

```diff
@@ -160,7 +162,11 @@ define(function (require, exports, module) {
         Menus.getMenu(Menus.AppMenuBar.VIEW_MENU).addMenuItem(CMD_TOGGLE_PURE_CODE, "", Menus.AFTER, CMD_TOGGLE_PANELS);
 
         KeyBindingManager.addBinding(CMD_TOGGLE_PURE_CODE, [ {key: togglePureCodeKey}, {key: togglePureCodeKeyMac, platform: "mac"} ]);
+
+        //default toggle panel shortcut was ctrl+shift+` as it is present in one vertical line in the keyboard. However, we later learnt
```

Comment:
Nitpick: This comment could be something like "The default "Toggle Panels" shortcut was CTRL+SHIFT+` as it is present in one vertical line in the standard US keyboard. However, we later learned from the IQE team that some non-US keyboards do not have an dedicated `-key, so an additional shortcut "CTRL+SHIFT+1" was added"